### DISCUSSION TITLE
I want to setup teslamate on a vps and access direct from the public IP

### DISCUSSION THREAD

**Discussion by Hxd097** [teslamate-org/teslamate#1661] [Getting started]
I have been reading this guide to setup teslamate on VPS  
[https://docs.teslamate.org/docs/guides/traefik](https://docs.teslamate.org/docs/guides/traefik)  
but I don't have a public FQDN and my vps has problem support FQDN,；  
So I just want to access teslamate via IP, How should I change the config to make this work?

**Comment by jun3280net** [comment]
> I have been reading this guide to setup teslamate on VPS
> https://docs.teslamate.org/docs/guides/traefik
> but I don't have a public FQDN and my vps has problem support FQDN,；
> So I just want to access teslamate via IP, How should I change the config to make this work?

Hi @Hxd097 - I wrote a guide here : [https://github.com/adriankumpf/teslamate/issues/784](https://github.com/adriankumpf/teslamate/issues/784) which might be able provide some insight.  The first time I setup Teslamate, I had created a VPS as well, but ran into some issues trying to implement FQDN as well.   I ended up going a completely different route - but works well for my needs.

That said you will need some sort of a Proxy Manager to translate the FQDN to the specific IP Addreess I think.

**Reply by Hxd097** [reply]
thanks mate， I ended up setting teslamate on a raspberrypi works only at home but fine.
